allennlp/commands/fine_tune.py

```diff
@@ -215,12 +215,20 @@ def fine_tune_model(model: Model,
     for name in tunable_parameter_names:
         logger.info(name)
 
-    trainer = Trainer.from_params(model,
-                                  serialization_dir,
-                                  iterator,
-                                  train_data,
-                                  validation_data,
-                                  trainer_params)
+    # If the config specifies a trainer subclass, we should use it.
+    # There must exist a trainer.
+    trainer_params = params.get("trainer")
```

Comment:
`trainer_params` is already popped above, so this is returning `None`.  You can just remove this line, and the comments above don't really add anything, so I'd probably remove those too.

Reply:
My bad. I didn't notice the line. I will fix it.